src/frontend/components/StudentViewersList/index.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Box, List } from 'grommet';
+import { defineMessages, useIntl } from 'react-intl';
+
+import {
+  useParticipantsStore,
+  ParticipantType,
+} from 'data/stores/useParticipantsStore/index';
+import { StudentViewersListHeader } from './StudentViewersListHeader';
+import { StudentViewersListItem } from './StudentViewersListItem';
+
+const messages = defineMessages({
+  onStage: {
+    defaultMessage: 'On stage',
+    description: 'On-stage users are displayed under this label.',
+    id: 'components.ViewersList.onStage',
+  },
+  otherViewers: {
+    defaultMessage: 'Other participants',
+    description: 'Connected users are displayed under this label.',
```

Comment:
```suggestion
    description: 'Connected participants are displayed under this label.',
```

Reply:
done in [e02c29134a3b2b55b074b7787123df0695ddc6df](https://github.com/openfun/marsha/pull/1282/commits/e02c29134a3b2b55b074b7787123df0695ddc6df)